rdm-schema.json

```
@@ -9,11 +9,27 @@
     "manufacturer_id": {
       "$comment": "Manufacturer IDs are assigned by ESTA and are commonly expressed in hexadecimal. Users may expect to see these values in the UI as hexadecimal.",
       "title": "Manufacturer ID",
-      "description": "The manufacturer ID.",
+      "description": "The Manufacturer ID.",
       "type": "integer",
       "minimum": 0,
       "maximum": 65535
     },
+    "device_model_id": {
+      "$comment": "The Device Model ID is a 16-bit value determined by the manufacturer. Users may expect to see these values in the UI as hexadecimal.",
+      "title": "Device Model ID",
+      "description": "The Device Model ID.",
+      "type": "integer",
+      "minimum": 0,
+      "maximum": 65535
+    },
+    "software_version_id": {
+      "$comment": "The Software Version ID is a 32-bit value determined by the manufacturer.",
```

It's not just a random 32-bit value they picked:
```suggestion
      "$comment": "The Software Version ID is a 32-bit value for a particular software release determined by the manufacturer.",
```